modules/rotating-certificate-authority.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// security/certificate_types_descriptions/etcd-certificates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rotating-certificate-authority_{context}"]
+== Rotating the etcd certificate
```

Comment:
```suggestion
= Rotating the etcd certificate
```
The first heading in a module is always an H1. Adjust the leveloffset in the assembly, if necessary.